mw-webapp/src/component/accordion/Accordion.cy.tsx

```diff
@@ -0,0 +1,101 @@
+import {Accordion, accordionTypes} from "src/component/accordion/Accordion";
+import {AccordionContent} from "src/component/accordion/AccordionContent/AccordionContent";
+import {AccordionTrigger} from "src/component/accordion/AccordionTrigger/AccordionTrigger";
+import {getDataCy} from "src/helpers/getDataCy";
+
+describe("Accordion component", () => {
+  const FIRST_TRIGGER = "trigger1";
+  const SECOND_TRIGGER = "trigger2";
+  const FIRST_CONTENT = "content1";
+  const SECOND_CONTENT = "content2";
+
+  const ACCORDION_ITEMS_EXAMPLE = [
+    {
+      trigger: <AccordionTrigger
+        text={FIRST_TRIGGER}
+        dataCy={FIRST_TRIGGER}
+      />,
+      content: <AccordionContent
+        content={<div>
+          {FIRST_CONTENT}
+        </div>}
+        dataCy={FIRST_CONTENT}
+      />,
+    },
+    {
+      trigger: <AccordionTrigger
+        text={SECOND_TRIGGER}
+        dataCy={SECOND_TRIGGER}
+      />,
+      content: <AccordionContent
+        content={<div>
+          {SECOND_CONTENT}
+        </div>}
+        dataCy={SECOND_CONTENT}
+      />,
+    },
+  ];
+
+  it("should render the accordion and all options", () => {
+    cy.mount(<Accordion items={ACCORDION_ITEMS_EXAMPLE} />);
```

Comment:
DRY. I think y can create function (example mountAccordion) and use it for mount accordion

Reply:
Done.